ngraph/frontend/onnx_import/src/core/node.cpp

```diff
@@ -207,6 +219,13 @@ namespace ngraph
             return m_pimpl->has_attribute(name);
         }
 
+        Subgraph Node::get_subgraph_from_attribute(
```

Comment:
Let's check if we can name this function `get_attribute_value` to keep in line with the others.

Reply:
Using this overloaded function would look like: get_attribute_value(map). In this case (without <Subgraph>) it can confuse the programmer what the function is actually supposed to do and return. To be in line with others it would have to be a template, but it doesn't make sense for other types of attributes.

What do You think?
@tsocha @mbencer @postrational @tomdol

Reply:
To me `get_subgraph_from_attribute` is fine.